src/components/ContentPresenter.js

```diff
@@ -0,0 +1,66 @@
+import React, {Component} from "react";
+import I from "immutable";
+
+
+const smallImage = require("/Users/alibilgi/Desktop/GithubBadge/public/styles/images/small.png");
+
+// Initial response objects
+let repos = [{}];
+let users = {
+  message: ""
+};
+
+class ContentContainer extends Component {
+
+
+  render() {
+
+    if (!I.List.isList(this.props.responseData)) {
+      repos = JSON.parse(this.props.responseData[0]);
+      users = JSON.parse(this.props.responseData[1]);
+    } else {
+      repos = [{}];
+      users = {
+        message: ""
+      };
+    }
+
+    return (
+        <div className="content">
+            <div className="col1">
+                <div className="row_of_col1">
+                    <a href={users.html_url || "https://github.com/404"}>
+                        <img src={users.avatar_url || "https://avatars2.githubusercontent.com/u/5779565?v=3&s=88"} alt="empty" />
+                    </a>
+                </div>
+                <div className="row_of_col1" />
+                <div className="row_of_col1"><img src={smallImage} alt="empty"/></div>
+            </div>
+            <div className="col2">
+                <div className="row_of_col2">
+                    <span className="topOnes">
+                        <a href={users.html_url || "https://github.com/404"}>
+                            {users.login || users.message }
+                        </a>
+                    </span>
+                </div>
+                <div className="row_of_col2">
+                    <span className="topOnes">
+                        {repos[0] ? repos[0].name : "no recent repo activity"} {repos[0] ? "(" + repos[0].language + ")" : ""}
```

Comment:
because I used promise.all() method while requesting to server, and as a result i'm getting an array with sized two.